### PR TITLE
Ansible tower sonar deploy

### PR DESCRIPTION
**Pull request by shay6**
waiting for code review

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
-roles/
+#roles/
```
Comment (abraverm) [inline .gitignore]: remove the line

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
 Install required roles by:
 
   ansible-galaxy install -r requirements.yml -p roles
```
Comment (abraverm) [inline README.md]: `ansible-galaxy install -r roles/requirements.yml -p roles` and then you don't need the comment in lines 4-7, and you won't have duplicate requirements.yml files

File: playbook.yml [hunk 3]
```diff
@@ -22,8 +22,9 @@
     when: sonar == "db"
```
Comment (abraverm) [inline playbook.yml:23]: Please explain this part with a comment. And if its a bug, then please report to Ansible and add it here.

File: rhel.yml [hunk 4]
```diff
@@ -25,3 +25,5 @@
     name: "{{ item }}"
   with_items:
   - unzip
+  - python-psycopg2
+  - MySQL-python
```
Comment (abraverm) [inline rhel.yml]: We are deploying postgresql database, why do you need MySQL library?

File: roles/requirements.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (abraverm) [inline roles/requirements.yml]: please remove ./requirements.yml

File: playbook.yml (hunk 3 above)
Comment (shay6) [reply playbook.yml:23]: waiting for permissions to open the bug on Bugzilla

File: roles/requirements.yml (hunk 5 above)
Comment (shay6) [reply roles/requirements.yml]: done

**Comment by abraverm** [comment]
Looks good, I will merge it once we have BZ.

**Pull request closed without merging**